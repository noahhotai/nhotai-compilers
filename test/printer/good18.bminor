testit: function boolean ( x: integer, y: integer ) = {

    	/* try the other way around */
    i:integer = 0;
	for(i=0;i<x;i++) {
            return 5; 
    }
}
