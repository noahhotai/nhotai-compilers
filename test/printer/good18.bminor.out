Program parsed successfully.
testit: function boolean (x : integer, y : integer) = {
    i: integer = 0;
    for(i=0;i<x;i++){
        return 5;
    }
}

